src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/Slider_themeresources.xaml

```diff
@@ -198,8 +198,7 @@
                                         <ControlTemplate TargetType="Thumb">
                                             <!-- Uno specific (LinearGradientBrush borders): We need additional Grid here #6457 -->
                                             <Grid>
-                                                <!-- Uno specific: WinUI uses Margin -2 here, which causes clipping for us. #16484 -->
-                                                <Border x:Name="ThumbBorder" Margin="0" Background="{ThemeResource SliderOuterThumbBackground}" BorderBrush="{ThemeResource ControlStrokeColorDefaultBrush}" BorderThickness="{TemplateBinding BorderThickness}" CornerRadius="{ThemeResource SliderThumbCornerRadius}">
+                                                <Border x:Name="ThumbBorder" Margin="-2" Background="{ThemeResource SliderOuterThumbBackground}" BorderBrush="{ThemeResource ControlStrokeColorDefaultBrush}" BorderThickness="{TemplateBinding BorderThickness}" CornerRadius="{ThemeResource SliderThumbCornerRadius}">
```

Comment:
Have you confirmed whether Android and iOS work with this?

Reply:
Yes, everything seems to be fine 🤔